README.md

```diff
@@ -132,10 +132,25 @@ If you wish, you can manually delegate exceptions, e.g. `try { ... } catch(e) {
 
 ## Setup for ReactJS
 
-Follow the general instructions denoted in _Setup for JavaScript_ to load and initialize the library.  
+Follow the general instructions denoted in _Setup for JavaScript_ to load and initialize the library.
 
 There is nothing specific that needs to be done with React, other than making sure to initialize the library in your root entry point(typically `index.js`).
 
+## Configuring without an API key
+
+If you are in a situation where an API key is not an option, you can configure the endpoint that errors are sent to with the following:
```

Comment:
Instead of "you can configure the endpoint that errors are sent to". You should also explain what's the idea:
"you can forward these errors to an endpoint on your server, and report them server-side. You can can configure the endpoint that errors are sent to...."